masonry/src/event.rs

```diff
@@ -181,65 +187,105 @@ impl From<PointerButton> for PointerButtons {
 // TODO - How can RenderRoot express "I started a drag-and-drop op"?
 // TODO - Touchpad, Touch, AxisMotion
 // TODO - How to handle CursorEntered?
+/// A pointer-related event.
+///
+/// A pointer in this context can be a mouse, a pen, a touch screen, etc. Though
+/// Masonry currently doesn't really support multiple pointers.
 #[derive(Debug, Clone)]
 pub enum PointerEvent {
+    /// A pointer was pressed.
     PointerDown(PointerButton, PointerState),
+    /// A pointer was released.
     PointerUp(PointerButton, PointerState),
+    /// A pointer was moved.
     PointerMove(PointerState),
+    /// A pointer entered the window.
     PointerEnter(PointerState),
+    /// A pointer left the window.
     PointerLeave(PointerState),
+    /// A mouse wheel event.
     MouseWheel(LogicalPosition<f64>, PointerState),
+    /// During a file drag-and-drop operation, a file was kept over the window.
     HoverFile(PathBuf, PointerState),
+    /// During a file drag-and-drop operation, a file was dropped on the window.
     DropFile(PathBuf, PointerState),
+    /// A file drag-and-drop operation was cancelled.
     HoverFileCancel(PointerState),
+    /// A pinch gesture was detected.
     Pinch(f64, PointerState),
```

Comment:
It would be helpful for users to document some of the data stored in the enum's parameters, like the meaning behind the MouseWheel event's logical position, and pinch's floating point value.

Reply:
I added some notes, though the "units" used will need to be better documented.